apps/omg_child_chain/lib/omg_child_chain/fee_parser.ex

```diff
@@ -39,28 +39,75 @@ defmodule OMG.ChildChain.FeeParser do
     end
   end
 
-  defp parse_fee_spec(%{"flat_fee" => fee, "token" => token}) do
+  defp parse_fee_spec(%{
```

Comment:
why string keys here?

Reply:
These keys come from reading the json file so they are strings. (also they were already strings before).